websockets.js

```diff
@@ -24,7 +24,7 @@ Sockets.regenerate = function (socket, data, callback) {
 		}
 
 		const baseUrl = nconf.get('url').replace(/.*?:\/\//g, '');
-		const issuer = encodeURIComponent(meta.config.title.replace(/\s/, '%20')).replace('+', '%20');
+		const issuer = encodeURIComponent(meta.config.title).replace('+', '%20');
 		const account = encodeURIComponent(userslug + '@' + baseUrl).replace('+', '%20');
```

Comment:
oh and btw. I don't think `encodeURIComponent` does ever encode spaces as `+` I think it always uses `%20`

Reply:
As per 068081cc148fe2c1dd23d623cddd680b1016e5d7, the replacement of `+` is done for iOS compatibility, so it should be left alone :smile:

Reply:
It defies logic, but only happens on iOS, so I will not tempt fate by removing it :grimacing:

Reply:
Probably a good call ^^